scdl/scdl.py

```diff
@@ -353,6 +353,13 @@ def download_all_of_a_page(tracks):
         logger.info('\nTrack n°{0}'.format(counter))
         download_track(track)
 
+def try_utime(path, filetime):
```

Comment:
The idea was inspired from [youtube-dl](https://github.com/rg3/youtube-dl/blob/1b6712ab2378b2e8eb59f372fb51193f8d3bdc97/youtube_dl/postprocessor/common.py#L58).

But in general using a try catch will prevent breaking change of your code. And like I said I did not test it on Windows.